3.evaluate_model/get_LOIO_probabilities_all_featurespaces_no_retraining.ipynb

```diff
@@ -0,0 +1,1258 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #62.            # create logistic regression model with following parameters
</pre><p>Same thing here, consider removing this code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/phenotypic_profiling_model/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
💯 thanks!